jbpm-process-svg/src/test/java/org/jbpm/process/svg/TestEvalutionSVG.java

```diff
@@ -109,26 +109,33 @@ public void testSubProcessLink() throws Exception {
 
     @Test
     public void testStunnerSubProcessLink() throws Exception {
-        String elementId = "_D254259B-8C23-4498-9BE1-995E2DC66726";
+        testSubProcessLink("_D254259B-8C23-4498-9BE1-995E2DC66726", "/stunner.parentP-svg.svg");
+    }
+
+    @Test
+    public void testBoundaryEventSubProcessLink() throws Exception {
+        testSubProcessLink("_86E33F6B-FD3B-4E07-8D60-D273A54FE6B7", "/svg-test.svg");
```

Comment:
please rename the test file to indicate purpose - see the file above is `stunner.parentP-svg.svg`, could this be `stunner.parentP.boundaryEvent-svg.svg` ?

Reply:
done